docs/docs/guides/automate/declarative-automation/customizing-automation-conditions/arbitrary-python-automation-conditions.md

```diff
@@ -1,9 +1,15 @@
 ---
 description: Define custom AutomationConditions in Dagster to execute arbitrary Python code to handle complex business logic.
-sidebar_position: 400
+sidebar_position: 600
 title: Arbitrary Python automation conditions
 ---
 
+:::note
+
+This functionality is experimental and may change in future releases.
```

Comment:
If this is Preview or Beta, you can replace this note with one of the lifecycle admonitions, e.g.:

```
import Preview from '@site/docs/partials/\_Preview.md';

<Preview />
```